tools/wpt/commands.json

```diff
@@ -0,0 +1,2 @@
+{"run": {"path": "run.py", "script": "run", "parser": "create_parser", "help": "Run tests in a browser",
+         "virtualenv": true, "install": ["requests"], "requirements": ["../wptrunner/requirements.txt"]}}
```

Comment:
If the "requests" module is necessary to use the `wptrunner` module, why isn't it listed in `wptrunner/requirements.txt`?